### PR TITLE
Change list-headers to view the chainheaders instead

### PR DESCRIPTION
**Pull request by SWvheerden**
## Description

The list-headers command used to display the headers. Some of the detail was removed from the headers such as the target_difficulty.  This PR will now displays the chain_header which still has this detail in it. 
Adds in an option to printout in csv, some raw header deatails 

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [x] I'm merging against the `development` branch.
* [x] I ran `cargo-fmt --all` before pushing.
* [x] I ran `cargo test` successfully before submitting my PR.
* [x] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: base_layer/core/src/chain_storage/accumulated_data.rs [hunk 1]
```diff
@@ -278,12 +279,40 @@ pub struct BlockHeaderAccumulatedData {
...
+        writeln!(f, "Achieved difficulty: {}", self.achieved_difficulty)?;
+        writeln!(f, "Total accumulated difficulty: {}", self.total_accumulated_difficulty)?;
+        writeln!(
+            f,
+            "Accumulated monero difficulty: {}",
+            self.accumulated_monero_difficulty
+        )?;
+        writeln!(
+            f,
+            "Accumulated monero difficulty: {}",
```
Comment (delta1) [inline base_layer/core/src/chain_storage/accumulated_data.rs]: Duplicate “Monero”?

File: base_layer/core/src/chain_storage/accumulated_data.rs [hunk 2]
```diff
@@ -278,12 +279,40 @@ pub struct BlockHeaderAccumulatedData {
...
+            self.accumulated_blake_difficulty
```
Comment (delta1) [inline base_layer/core/src/chain_storage/accumulated_data.rs]: Should this be SHA instead of Blake?

Comment (SWvheerden) [reply base_layer/core/src/chain_storage/accumulated_data.rs]: No, its a legacy variable name. I believe refactoring this is beyond the scope of this PR.

Comment (delta1) [reply base_layer/core/src/chain_storage/accumulated_data.rs]: Oh yeah, no problem.

**Comment by stringhandler** [comment]
Looks fine, clippy CI failed though

**Pull request merged**